django_loci/static/django-loci/js/floorplan-widget.js

```diff
@@ -31,7 +31,7 @@
         map.setView([0, 0], 0);
 
         function updateInput(e) {
-            var latlng = e.latlng || e.target._latlng;
+            var latlng = e.latlng || e.target.latlng;
```

Comment:
not sure this is ok, does leaflet offer this attribute? If yes please show reference to existing practices/docs
Have you tested it?

Reply:
so this should not be changed, but add a new configuration to .jslintrc?